dashboards-to-provision/services.json

```diff
@@ -1639,7 +1639,7 @@
     "list": []
   },
   "time": {
-    "from": "now-3h",
+    "from": "now-7d",
```

Comment:
You're sure?

Reply:
Probably a personal preference, but I like to have a 1-week overview... We can take the inverse geometric mean of mine, yours, and @cgrigis preference ;)

@cgrigis what is your preference for the time-frame of stats.c4dt.org?

Reply:
I would personally put the last 24h :smile:
Then again it is easily adjustable with the dropdown, and 3h possibly loads faster?

Reply:
So:

```
(3 * 24 * 168) ^ 1/3 = 23
```

I'd say we round up to 24 hours... So the geometric mean of 3 and 168 is about 24h.